srai/regionizers/voronoi_regionizer.py

```diff
@@ -0,0 +1,68 @@
+"""
+Voronoi Regionizer.
+
+This module contains voronoi regionizer implementation.
+"""
+
+from itertools import combinations
+
+import geopandas as gpd
+
+from ._spherical_voronoi import generate_voronoi_regions
+
+
+class VoronoiRegionizer:
+    """A Voronoi regionizer."""
+
+    def __init__(self, seeds: gpd.GeoDataFrame, max_meters_between_points: int = 10_000) -> None:
+        """
+        VoronoiRegionizer.
+
+        Voronoi [1] regionizer allows the given geometries to be divided into
+        Thiessen polygons using geometries that are the seeds. To minimize distortions
+        tessellation will be performed on a sphere using SphericalVoronoi [2] function
+        from scipy library.
+
+        All (multi)polygons from seeds GeoDataFrame will be transformed to their centroids,
+        because scipy function requires only points as an input.
+
+        Seeds laying on a single arc might result in a ValueError exception.
+
```

Comment:
[Please use `Raises` section of the Google docstring](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html#:~:text=Raises%3A%0A%20%20%20%20%20%20%20%20AttributeError%3A%20The%20%60%60Raises%60%60%20section%20is%20a%20list%20of%20all%20exceptions%0A%20%20%20%20%20%20%20%20%20%20%20%20that%20are%20relevant%20to%20the%20interface.%0A%20%20%20%20%20%20%20%20ValueError%3A%20If%20%60param2%60%20is%20equal%20to%20%60param1%60.)